tracing-subscriber/src/fmt/time/time_crate.rs

```diff
@@ -180,6 +197,22 @@ where
 // === impl UtcTime ===
 
 impl UtcTime<well_known::Rfc3339> {
+    /// Returns a formatter that formats the current [UTC time] in the
+    /// [RFC 3339] format, which is a subset of the [ISO 8601] timestamp format,
+    /// but with a variable number of decimal digits.
+    ///
+    /// Use [`UtcTime::iso_8601`] instead.
+    ///
+    /// [local time]: https://docs.rs/time/0.3/time/struct.OffsetDateTime.html#method.now_utc
+    /// [RFC 3339]: https://datatracker.ietf.org/doc/html/rfc3339
+    /// [ISO 8601]: https://en.wikipedia.org/wiki/ISO_8601
+    #[deprecated(note = "this has a variable number of decimal digits, use `UtcTime::iso_8601()` instead")]
+    pub fn rfc_3339() -> Self {
+        Self::new(well_known::Rfc3339)
+    }
+}
+
+impl UtcTime<well_known::Iso8601> {
     /// Returns a formatter that formats the current [UTC time] in the
     /// [RFC 3339] format, which is a subset of the [ISO 8601] timestamp format.
```

Comment:
similarly, this function returns the ISO 8601 formatter:
```suggestion
    /// Returns a formatter that formats the current [UTC time] in the
    /// the [ISO 8601] timestamp format.
```